src/view/auth/AuthContainer.tsx

```diff
@@ -1,27 +1,39 @@
-import React, { useState, useEffect } from "react";
+import React, { useEffect } from "react";
 import { AuthForm } from "./AuthForm";
 import { API_BASE_URL } from "../../controller/config";
 import RestAPIClient from "../../api/RestAPIClient";
+import { RootState } from "../../redux/create";
+import { Dispatch, AnyAction, bindActionCreators } from "redux";
+import { appStatusActionCreators } from "../../redux/modules/appStatus";
+import { connect } from "react-redux";
+import { LoadingPage } from "../layout/Loading";
 
-type AuthContext = {
-  loggedIn: boolean;
-  setLoggedIn: React.Dispatch<boolean>;
-  name: string;
-};
+const mapStateToProps = (state: RootState) => ({
+  user: state.appStatus.user,
+  isInitializing: state.appStatus.isInitializing
+});
+
+const mapDispatchToProps = (dispatch: Dispatch<AnyAction>) =>
+  bindActionCreators(
+    {
+      setUser: appStatusActionCreators.setUser,
+      setIsInitializing: appStatusActionCreators.setIsInitializing
+    },
+    dispatch
+  );
 
-interface AuthContainerProps {
+interface AuthContainerProps extends React.Props<any> {
```

Comment:
This is a workaround for a react-redux type issue; we should fix this before merging. Might require manually adding `children` to `AuthContainerProps`